datahub-web-react/package.json

```diff
@@ -67,7 +67,7 @@
         "react-color": "^2.19.3",
         "react-dom": "^17.0.0",
         "react-email-share-link": "^1.0.3",
-        "react-helmet": "^6.1.0",
+        "react-helmet-async": "^1.3.0",
```

Comment:
This upgrade resolves a js console warning and the old package react-helmet recommends upgrading to react-helmet-async.